### ISSUE TITLE
Game Crashes Around a Minute In

### ISSUE THREAD

**Issue by Yatotoro** [Lunatrius/Schematica#361]
Game crashes around a minute in, both multiplayer and singleplayer
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  BetterSprintingCore (BetterSprinting++MC-1.12.2++v2.1.1.jar)
Contact their authors BEFORE contacting forge

// This is a token for 1 free hug. Redeem at your nearest Mojangsta: [~~HUG~~]

Time: 7/16/18 11:20 PM
Description: Unexpected error

java.nio.BufferOverflowException
	at java.nio.Buffer.nextPutIndex(Buffer.java:521)
	at java.nio.DirectIntBufferU.put(DirectIntBufferU.java:297)
	at net.optifine.render.VboRegion.drawArrays(VboRegion.java:299)
	at net.minecraft.client.renderer.vertex.VertexBuffer.func_177358_a(VertexBuffer.java:50)
	at com.github.lunatrius.schematica.client.renderer.chunk.container.SchematicChunkRenderContainerVbo.func_178001_a(SchematicChunkRenderContainerVbo.java:28)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderBlockLayer(RenderSchematic.java:802)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.func_174977_a(RenderSchematic.java:793)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderWorld(RenderSchematic.java:354)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderSchematic(RenderSchematic.java:278)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.onRenderWorldLast(RenderSchematic.java:247)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_28_RenderSchematic_onRenderWorldLast_RenderWorldLastEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.client.ForgeHooksClient.dispatchRenderLast(ForgeHooksClient.java:168)
	at sun.reflect.GeneratedMethodAccessor97.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.optifine.reflect.Reflector.callVoid(Reflector.java:653)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1946)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1613)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1384)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1117)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at java.nio.Buffer.nextPutIndex(Buffer.java:521)
	at java.nio.DirectIntBufferU.put(DirectIntBufferU.java:297)
	at net.optifine.render.VboRegion.drawArrays(VboRegion.java:299)
	at net.minecraft.client.renderer.vertex.VertexBuffer.func_177358_a(VertexBuffer.java:50)
	at com.github.lunatrius.schematica.client.renderer.chunk.container.SchematicChunkRenderContainerVbo.func_178001_a(SchematicChunkRenderContainerVbo.java:28)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderBlockLayer(RenderSchematic.java:802)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.func_174977_a(RenderSchematic.java:793)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderWorld(RenderSchematic.java:354)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderSchematic(RenderSchematic.java:278)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.onRenderWorldLast(RenderSchematic.java:247)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_28_RenderSchematic_onRenderWorldLast_RenderWorldLastEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.client.ForgeHooksClient.dispatchRenderLast(ForgeHooksClient.java:168)
	at sun.reflect.GeneratedMethodAccessor97.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.optifine.reflect.Reflector.callVoid(Reflector.java:653)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1946)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1613)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['Yatotoro'/4413715, l='MpServer', x=-9748.04, y=67.00, z=5617.28]]
	Chunk stats: MultiplayerChunkCache: 82, 82
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-45243,68,28064), Chunk: (at 5,4,0 in -2828,1754; contains blocks -45248,0,28064 to -45233,255,28079), Region: (-89,54; contains chunks -2848,1728 to -2817,1759, blocks -45568,0,27648 to -45057,255,28159)
	Level time: 128067 game time, 469733984 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 7 total; [EntityPlayerSP['Yatotoro'/4413715, l='MpServer', x=-9748.04, y=67.00, z=5617.28], EntityMinecartChest['Minecart with Chest'/4413763, l='MpServer', x=-9714.50, y=37.06, z=5584.50], EntityChicken['Chicken'/4413762, l='MpServer', x=-9755.27, y=69.00, z=5634.74], EntityChicken['Chicken'/4413761, l='MpServer', x=-9758.17, y=68.00, z=5635.49], EntityMinecartChest['Minecart with Chest'/4413760, l='MpServer', x=-9722.50, y=28.06, z=5610.50], EntityMinecartHopper['Minecart with Hopper'/4413759, l='MpServer', x=-9734.50, y=68.00, z=5616.50], EntityZombie['Zombie'/4414366, l='MpServer', x=-9713.50, y=28.00, z=5597.50]]
	Retry entities: 0 total; []
	Server brand: BungeeCord (git:BungeeCord-Bootstrap:1.12-SNAPSHOT:2356dbc:1318) <- Paper
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:532)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2738)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:426)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 4318243736 bytes (4118 MB) / 5138022400 bytes (4900 MB) up to 5138022400 bytes (4900 MB)
	JVM Flags: 26 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx5G -Xms5G -Xmn2200m -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:+UseNUMA -XX:+CMSParallelRemarkEnabled -XX:MaxTenuringThreshold=15 -XX:MaxGCPauseMillis=30 -XX:GCPauseIntervalMillis=150 -XX:+UseAdaptiveGCBoundary -XX:-UseGCOverheadLimit -XX:+UseBiasedLocking -XX:SurvivorRatio=8 -XX:TargetSurvivorRatio=90 -XX:MaxTenuringThreshold=15 -XX:+UseFastAccessorMethods -XX:+UseCompressedOops -XX:+OptimizeStringConcat -XX:+AggressiveOpts -XX:ReservedCodeCacheSize=2048m -XX:+UseCodeCacheFlushing -XX:SoftRefLRUPolicyMSPerMB=10000 -XX:ParallelGCThreads=10
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.42 Powered by Forge 14.23.4.2705 Optifine OptiFine_1.12.2_HD_U_E1 8 mods loaded, 8 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID              | Version      | Source                                      | Signature                                |
	|:------ |:--------------- |:------------ |:------------------------------------------- |:---------------------------------------- |
	| UCHIJA | minecraft       | 1.12.2       | minecraft.jar                               | None                                     |
	| UCHIJA | mcp             | 9.42         | minecraft.jar                               | None                                     |
	| UCHIJA | FML             | 8.0.99.99    | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJA | forge           | 14.23.4.2705 | forge-1.12.2-14.23.4.2705.jar               | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| UCHIJA | bettersprinting | 2.1.1        | BetterSprinting++MC-1.12.2++v2.1.1.jar      | None                                     |
	| UCHIJA | lunatriuscore   | 1.2.0.42     | LunatriusCore-1.12.2-1.2.0.42-universal.jar | None                                     |
	| UCHIJA | pi              | 4.0.0        | Potion+Icon+Remover-1.12-4.0.0.jar          | None                                     |
	| UCHIJA | schematica      | 1.8.0.169    | Schematica-1.12.2-1.8.0.169-universal.jar   | None                                     |

	Loaded coremods (and transformers): 
BetterSprintingCore (BetterSprinting++MC-1.12.2++v2.1.1.jar)
  chylex.bettersprinting.system.core.TransformerEntityPlayerSP
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 390.77' Renderer: 'GeForce 840M/PCIe/SSE2'
	Launched Version: 1.12.2-forge1.12.2-14.23.4.2705
	LWJGL: 2.9.4
	OpenGL: GeForce 840M/PCIe/SSE2 GL version 4.6.0 NVIDIA 390.77, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: §f§lNotroDan Overlay Pack.zip (incompatible)
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 4x Intel(R) Core(TM) i7-5500U CPU @ 2.40GHz
	The 5zig Mod Version: 3.12.5
	The 5zig Mod Plugins: []
	Forge: true
	GUI: ~~NULL~~
	OptiFine Version: OptiFine_1.12.2_HD_U_E1
	OptiFine Build: 20180628-111228
	Render Distance Chunks: 32
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 390.77
	OpenGlRenderer: GeForce 840M/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 4
```

**Comment by Lunatrius** [comment]
Did you have a schematic loaded in?

**Comment by Yatotoro** [comment]
Found the problem, was using this with other mods, apparently dosent work with it. Just run the core and schematica and works fine

**Comment by Pokechu22** [comment]
This is a duplicate of #321, caused by Optifine's Render Regions setting.
